tasks/release.py

```diff
@@ -787,9 +788,9 @@ def cleanup(ctx):
 @task
 def check_omnibus_branches(ctx):
     base_branch = _get_release_json_value('base_branch')
-    if base_branch == 'main':
-        omnibus_ruby_branch = 'datadog-5.5.0'
-        omnibus_software_branch = 'master'
+    if base_branch == get_default_branch():
+        omnibus_ruby_branch = DEFAULT_BRANCHES['omnibus-ruby']
```

Comment:
question: this looks like we are always using the DEFAULT_BRANCHES for "main", is this intentional when we have the default branch as Agent 6?